episodes/09-open-and-plot-vector-layers.Rmd

```diff
@@ -0,0 +1,196 @@
+---
+title: "Open and Plot Vector Layers"
+teaching: 25
+exercises: 5
+---
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- How can you read, examine and visualize point, line and polygon vector data in R?
```

Comment:
We need to agree on whether it is second person, first person singular or first person plural and use the same in all episodes. Now we have all three options. I am okay with any version. Any reason why you thin first-person singular is better?

Reply:
I see now your explanation above. We can rephrase questions and objectives in all lessons as you suggest.

Reply:
Yes, as I explained above, I think that first person works better in questions - either singular or plural. I saw it was singular in the other three episodes in Part 2, so I changed it here into 'I' for consistency. But if you prefer 'we' instead of 'I', that sounds good as well - maybe even better. Only in objectives, I would use second person - 'you'.